src/Symfony/Component/Cache/Tests/Traits/RedisTraitTest.php

```diff
@@ -14,15 +14,11 @@
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Cache\Traits\RedisTrait;
 
+/**
+ * @requires extension redis
+ */
 class RedisTraitTest extends TestCase
 {
-    public static function setUpBeforeClass(): void
-    {
-        if (!getenv('REDIS_CLUSTER_HOSTS')) {
-            self::markTestSkipped('REDIS_CLUSTER_HOSTS env var is not defined.');
```

Comment:
but we still need the env var to run all the other test cases, don't we?

Reply:
Yes, but this env var presence is already checked in `testCreateConnection` so it can be removed from here